src/askui/agent_base.py

```diff
@@ -507,6 +509,101 @@ def locate_all(
         )
         return self._locate(locator=locator, screenshot=screenshot, model=model)
 
+    @telemetry.record_call(exclude={"locator", "screenshot"})
+    @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
+    def locate_all_elements(
+        self,
+        screenshot: Optional[InputSource] = None,
+        model_composition: ModelComposition | None = None,
+    ) -> list[DetectedElement]:
+        """Locate all elements in the current screen using AskUI Models.
+
+        Args:
+            screenshot (InputSource | None, optional): The screenshot to use for
+                locating the elements. Can be a path to an image file, a PIL Image
+                object or a data URL. If `None`, takes a screenshot of the currently
+                selected display.
+            model_composition (ModelComposition | None, optional): The model composition
+                 to be used for locating the elements.
+
+        Returns:
+            list[DetectedElement]: A list of detected elements
+
+        Example:
+            ```python
+            from askui import VisionAgent
+
+            with VisionAgent() as agent:
+                detected_elements = agent.locate_all_elements()
+                print(f"Found {len(detected_elements)} elements: {detected_elements}")
+            ```
+        """
+        _screenshot = load_image_source(
+            self._agent_os.screenshot() if screenshot is None else screenshot
+        )
+        return self._model_router.locate_all_elements(
+            image=_screenshot, model=model_composition or ModelName.ASKUI
+        )
+
+    @telemetry.record_call(exclude={"screenshot", "output_directory"})
+    @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
+    def annotate(
+        self,
+        screenshot: InputSource | None = None,
+        model_composition: ModelComposition | None = None,
+        output_directory: str = "reports",
```

Comment:
can we switch the params. I tried to move the model parameter always to the latest place. and rename it to `model`. I aggree it's only a model_composition but I would prefere here consitent naming.

```suggestion
        output_directory: str = "reports",
        model: ModelComposition | None = None,
```

Reply:
You missed the renaming of `mode_composition` to `model`